### PR TITLE
RA-1061 adding angular in progress

### PR DESCRIPTION
**Pull request by AdamGrzybkowski**

### CODE REVIEW

File: app/js/controllers.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+var conceptDirControllers = angular.module('conceptDirControllers', []);
```
Comment (rkorytkowski) [inline app/js/controllers.js:1]: Let's rename it to conceptDictControllers.

File: app/js/services.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+var conceptDirServices = angular.module('conceptDirServices', ['ngResource']);
+
+conceptDirServices.factory('Classes', ['$resource',
+                                   function($resource){
+                                     return $resource(
+                                    		 'http://localhost:8080/openmrs/ws/rest/v1/conceptclass/8d4907b2-c2cc-11de-8d13-0010c6dffd0f', {}, {
```
Comment (rkorytkowski) [inline app/js/services.js:6]: Please have a look at https://github.com/openmrs/openmrs-module-uicommons/blob/master/omod/src/main/webapp/resources/scripts/services/conceptService.js

All we need for the URL is var OPENMRS_CONTEXT_PATH = "openmrs". Let's define it as a global variable for now in services.js. We can safely assume that the page will be always served by the OpenMRS server so we do not have to specify the hostname and port in REST urls.

File: app/js/services.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+var conceptDirServices = angular.module('conceptDirServices', ['ngResource']);
```
Comment (rkorytkowski) [inline app/js/services.js:1]: Where does Dir come from? It would be clearer for me to name it conceptDictServices.

**Comment by rkorytkowski** [comment]
It looks good. I'll merge it at this point, but please respond to my comments by either fixing the code or replying to me in comments.

File: app/js/controllers.js (hunk 1 above)
Comment (AdamGrzybkowski) [inline app/js/controllers.js:1]: Done

File: app/js/services.js (hunk 2 above)
Comment (AdamGrzybkowski) [inline app/js/services.js:6]: Please see the new commit, should work now for both local server and OpenMRS server

**Pull request merged**